scripts/genappimage.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+########################################################################
+# Package the baries built on Travis-CI as an AppImage
+# By Simon Peter 2016
+# For more information, see http://appimage.org/
+########################################################################
+
+if [ -z "$ARCH" ]; then
+  export ARCH="$(arch)"
+fi
+
+APP=NVim
+LOWERAPP="${APP,,}"
+ROOT_DIR="$(git rev-parse --show-toplevel)"
+APP_BUILD_DIR="$ROOT_DIR/build"
+APP_DIR="$APP_BUILD_DIR/$APP.AppDir"
+
+# cd neovim
+GIT_REV="$(git rev-parse --short HEAD)"
+
+# Since neovim increments versions slower than vim, using
+# the commit's date makes more sense.
+VIM_VER="$(date -d "@$(git log -1 --format=%ct)" "+%F")"
+
+
+make install DESTDIR="$APP_DIR"
+
+# Move runtime from /usr/local to /usr
+mv "$APP_DIR"/usr/local/* "$APP_DIR/usr/"
+
+cd "$APP_BUILD_DIR"
+
+wget -q https://github.com/probonopd/AppImages/raw/master/functions.sh -O ./appimage_functions.sh
```

Comment:
Our other scripts all use curl. Though most portable would be using `cmake -DURL=https://github.com/probonopd/AppImages/raw/master/functions.sh -DFILE=./appimage_functions.sh cmake/Download.cmake`.